v3/src/components/data-display/data-display-utils.ts

```diff
@@ -73,87 +77,64 @@ export function handleClickOnCase(event: MouseEvent, caseID: string, dataset?: I
 
 export interface IMatchCirclesProps {
   dataConfiguration: IDataConfigurationModel
-  dotsElement: DotsElt
   pointRadius: number
   pointColor: string
   pointStrokeColor: string
   startAnimation: () => void
   instanceId: string | undefined
+  pixiPoints: PixiPoints
   animateChange?: boolean // default false
 }
 
 export function matchCirclesToData(props: IMatchCirclesProps) {
-  const {dataConfiguration, startAnimation, instanceId,
-      dotsElement, pointRadius, pointColor, pointStrokeColor, animateChange} = props,
-    id = dataConfiguration.id,
-    allCaseData = dataConfiguration.joinedCaseDataArrays,
-    caseDataKeyFunc = (d: CaseData) => `${d.plotNum}-${d.caseID}`,
-    circles = selectCircles(dotsElement, id),
-    duration = animateChange ? transitionDuration : 0
-  if (!circles) return
+  const { dataConfiguration, pixiPoints, startAnimation, pointRadius, pointColor, pointStrokeColor,
+          animateChange } = props
```

Comment:
Given that `transitionDuration` (above) and `animateChange` seem to no longer be used, it would seem that we've lost some animation capability. If that is true, my inclination would be to restore the definition of `duration` along with a TODO comment and ideally add a PT story if there isn't one already:
```typescript
  // TODO: restore animation configuration ability
  const duration = animateChange ? transitionDuration : 0 // eslint-disable-line @typescript-eslint/no-unused-vars